templates/careers/company-culture.html

```diff
@@ -0,0 +1,200 @@
+{% extends '/careers/base-tabs.html' %}
+{% block meta_copydoc %}https://docs.google.com/document/d/1PAeP_ZO4JzSBCi-fRp6F7SYtSSRY-4DcWLUf9HJaMxs/edit#heading=h.yys3mdcj4xua{% endblock meta_copydoc %}
+
+{% block title %}Company Culture | Careers{% endblock %}
+
+{% block meta_description %}Canonical prefers internal promotion and encourages high-performing colleagues to pursue diverse roles at the company over the course of their career, to develop a well-rounded perspective.{% endblock %}
+
+
+{% block content %}
+<div class="p-strip is-deep">
+  <div class="row">
+    <div class="col-start-large-4 col-8">
+      <h1 class="p-heading--2"><strong>Company culture</strong></h1>
+      <h2>We believe that talent is evenly distributed around the world.  Diversity is part of our strength. What unifies us isn’t our background,  it’s our mission to amplify open source.</h2>
+    </div>
+  </div>
+</div>
+
+<div class="p-strip u-no-padding--top">
+  <div class="row">
+    <hr class="u-no-margin--bottom">
+    <div class="col-medium-3 col-3">
+      <h3 class="p-muted-heading">We’re Canonical</h3>
+    </div>
+    <div class="col-medium-6 col-9">
+      <div class="row">
+        <div class="col-medium-2 col-3">
+          <h4 class="p-heading--2 u-data-border--top"><strong>950+</strong><br>Employees</h3>
+        </div>
+        <div class="col-medium-2 col-3">
+          <h4 class="p-heading--2 u-data-border--top"><strong>89</strong><br>Countries</h3>
+        </div>
+        <div class="col-medium-2 col-3">
+          <h4 class="p-heading--2 u-data-border--top"><strong>78</strong><br>Languages</h3>
+        </div>
+        <p>The future is already here, as open source. Canonical delivers it to the world. We play a critical role in broadening the benefits of open source to more people and more industries than ever before. This means bringing new work opportunities to people regardless of their location too.</p>
+      </div>
+    </div>
+  </div>
+</div>
+<div class="p-strip u-no-padding--top">
+  <div class="row">
+    <hr class="u-no-margin--bottom">
+    <div class="col-medium-2 col-3">
+      <h3 class="p-muted-heading">Fully distributed <br class="u-hide--small" />since 2004</h3>
+    </div>
+    <div class="col-medium-4 col-9">
+      <div class="row">
+        <div class="col-3 col-medium-1" style="margin-top: 0.5625rem">
```

Comment:
I would suggest trying to get the profile images and quotes side by side in small screens.